.gitignore

```diff
@@ -21,3 +21,8 @@ examples/strava-oauth/settings.cfg
 stravalib/tests/test.ini-example
 stravalib/tests/test.ini
 stravalib/docs/api
+
+# Flask app
+examples/strava-oauth/settings.cfg
+test.py 
+test-stravalib.py
```

Comment:
Are these supposed to be added locally?

Reply:
oh shoot - that was me trying to make sure i did not accidentally commit my test scripts to the repo. i get lazy sometimes and do `git add .` i can remove them from gitignore as i've moved the files outside of the stravalib directory so there is no worry about committing them.